code/perception/src/panoptic_segmentation/efficientps/semantic_head/semantic_head.py

```diff
@@ -0,0 +1,247 @@
+from math import ceil
+import torch
+import torch.nn as nn
+from torch.nn import functional as F
+from inplace_abn import InPlaceABN
+
+try:
+    from panoptic_segmentation. \
+        efficientps.utils.depthwise_separable_conv import \
+        DepthwiseSeparableConv
+except ImportError:
+    from efficientps.utils.depthwise_separable_conv import \
+        DepthwiseSeparableConv
+
+
+class SemanticHead(nn.Module):
+    """
+    Semantic Head compose of three main module DPC, LSFE and MC
```

Comment:
What does DPC, LSFE and MC stand for?

Reply:
Good question ^^ We copied this code from https://github.com/vincrichard/EfficientPS, so the shortcuts and Todos are not from us. And it is not important for our task :)